monash-human-power/MHP-website
# Update team 2023 S1

## Description



Update team for 2023 semester 1:

- Removed people who left the team end of last year
- Added new trikes from the last recruitment session
- Automatically sort team members within the subteam by position, and then by name

## Screenshots



## Steps to Test



Build as normal.

To test the sorting, add some new team members and see if it's sorted automatically
